resources/syntax/Jinja.sublime-syntax

```diff
@@ -322,6 +354,20 @@ contexts:
         - include: item_seperator
 
   logical_operators:
+    - match: '\b(is) +(not)?\b'
```

Comment:
I am not sure what the single space is supposed to mean ? Are you trying to match a single whitespace `\s+` ?

Reply:
It was supposed to match at least one space. `\s` is probably better to allow for tabs as well.